### PR TITLE
Align content within in the tables

### LINKED ISSUE #31: Fix current metrics design issues
- [x] drop left/right padding from "top 5" tables: PR #36 
- [x] hide overflow of long unit names with unbreakable words. PR #36 ; Reproducer:

      sudo systemd-run -p CPUQuota=8% -u calc-harder-aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.service dd if=/dev/urandom of=/dev/null
     sudo systemd-run -p CPUQuota=6% -u calc-harder-bbbbbb-ccccccc-ddddddd-eeeeeee-ffffff-gggggg-hhhhhhh-iiiii.service dd if=/dev/urandom of=/dev/null

  and after you are done:
 
      sudo systemctl stop calc-harder-aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.service calc-harder-bbbbbb-ccccccc-ddddddd-eeeeeee-ffffff-gggggg-hhhhhhh-iiiii.service

- [x] keep units on the same line as values PR #36
keepunit

- [ ] Pin the top 5 tables to the card bottom, so that they align vertically: PR #41

- [x] Show load values as proper key/values (needs issue #16 decided and fixed)

### PR DESCRIPTION
**Pull request by garrett**
I think this closes #37?

Fixes #31

### CODE REVIEW

**Comment by garrett** [comment]
Aha, I noticed an issue with the alignment of disks, as the string row is above the progress stack, instead of the other way around. I have an idea for an easy fix.

**Comment by martinpitt** [comment]
By and large this is great, and the CPU/memory tables are now aligned with and without swap.

I just noticed two things:

image

 * The "disks" mount points now get pushed to the bottom. At first sight I thought "uh, that's broken". Maybe it's intended, maybe it's not actually so bad. Should we restrict the flexing to the CPU/memory tables for now? Is that what you meant with your "alignment of disks" comment above?

 * The "network" table is misaligned with the other card contents. But this isn't actually a regression, current master already does that.

File: src/app.scss [hunk 1]
```diff
@@ -19,11 +19,30 @@
...
+
+      // Stretch the progress stack and description list so they fill up remaining space
+      .progress-stack,
+      .pf-c-description-list {
+          flex: auto;
+          align-content: start;
+      }
+
+      // Give the description list as much leftover space as possible,
+      // when it exists with other flexing elements
```
Comment (martinpitt) [inline src/app.scss:42]: Let's add a comment like "This will push the cgroups tables to the bottom, so that they become aligned".

**Comment by garrett** [comment]
Changes between the original and what's here are in 60c13a8. I have since force-pushed.

**Comment by garrett** [comment]
>    The "disks" mount points now get pushed to the bottom. At first sight I thought "uh, that's broken". Maybe it's intended, maybe it's not actually so bad. Should we restrict the flexing to the CPU/memory tables for now? Is that what you meant with your "alignment of disks" comment above?

Fixed. Yep. :wink:

>    The "network" table is misaligned with the other card contents. But this isn't actually a regression, current master already does that.

Due to `<th>`. I fixed this in the PR.

File: src/app.scss (hunk 1 above)
Comment (garrett) [reply src/app.scss]: Adjusted the comments to make this clearer.

**Comment by martinpitt** [comment]
Perfect, thank you! I just did a force-push to drop an excessive trailing space (that vim highlighted in bright red).

**Review by martinpitt — APPROVED** [review]

**Pull request merged**